src/akamai/js/akamaiHD_flash.js

```diff
@@ -546,54 +577,62 @@ require("../../../html5-common/js/utils/constants.js");
     }
 
     var raiseCaptionFound = function(event) {
-      var captionInfo = {
+        var captionInfo = {
               language: "CC",
               inStream: true,
               label: "In-Stream"
             };
       self.controller.notify(self.controller.EVENTS.CAPTIONS_FOUND_ON_PLAYING,captionInfo);
     }
 
-    // Receives a callback from Flash 
-    onCallback = _.bind(function(data) {
-      OO.log('[Akamai HD]:onCallback: ', data);
+    call = function() {
```

Comment:
Yes. "call" is referenced by flash. This function is the vital piece that sorts out which dom element to associate with which instantiation of a flash object, and then calls this.onCallback to execute it on only the proper element.